yosh/shell.py

```diff
@@ -40,11 +87,53 @@ def execute(cmd_tokens):
         signal.signal(signal.SIGINT, handler_kill)
         # Spawn a child process
         if platform.system() != "Windows":
-            # Unix support
-            p = subprocess.Popen(cmd_tokens)
-            # Parent process read data from child process
-            # and wait for child process to exit
-            p.communicate()
+            found = 0
+            std = 0
+            waitpid = 0
```

Comment:
In Python, let's use `True` and `False` for truth checking not `0` and `1` (except that there are more than 2 cases).